### DISCUSSION TITLE
How Hubble Law became "Hubble-Lemaitre" Law

### DISCUSSION THREAD

**Discussion by khuramonline** [a-cosmology-group/acg#36] [Metaphysics and Philosophy]
At that time I came across some proposals by some so-called historians of science that "Hubble Law" should be "Hubble-Lemaitre Law".

At that time, I had sent following email to some of those who had submitted such proposals. Off course, my efforts were of no avail. Just sharing may be for record:

> Sir,
> 
> It is with reference to your proposals regarding the renaming decision of Hubble Law to Hubble-Lemaître Law on following page:
> 
> https://arxiv.org/ftp/arxiv/papers/1909/1909.07731.pdf  
> 
> Sir you have excellently presented the case. However I could spot some missing links to the whole story.
> 
> I had one version of the story. Surely I also have found many missing links to my version as well ... by reading the details outlined by you.
> 
> Sir, you fully know about the translation (1931) discrepancies of that Lemaître's 1927 French article. Your analysis is ... however silent that in the translation 1931,  Lemaître , along with so-called "preliminary" things, also had omitted the reference to Hubble.
> 
> **That was reference to Hubble as a source of redshift-distance relationship.**
> 
> In year 1927, Lemaître knew about redshift-distance relationship out of two distinct sources.
> 
> 1- Firstly he knew about the redshift-distance relationship out of de-Sitter (1917) paper. Here Lemaître  might have misunderstood the actual de-Sitter as in 1917, Sitter had discussed redshift-distance relationship as a mere guess. Moreover, in 1930, de-Sitter and Eddington both were on same page that neither de-Sitter nor Einstein model was capable to account for newly observed redshift-distance relation.
> 
> 2- Secondly, Lemaître knew the method of how to empirically deduce distance of the galaxies out of a method which he took from Hubble himself. **Exact this reference was omitted from translation.**
> 
> There is more to this story which I have explained in my book (2018).
> 
> Following is PDF link to relevant part of the book:
> 
> https://zenodo.org/record/3530806#.Xh24qGpRXcs
> 
> And following is PDF link of complete book:
> 
> https://zenodo.org/record/3544779#.Xh24rGpRXct
> 
> Sir ........
> 
> Theory had only guess (1917) ... in year 1927 ,,, Lemaitre took that guess for real. He developed his equations keeping in view this factor. He also had learned to empirically deduce distance from redshift data that he had.
> 
> **He did reach to Hubble law before Hubble.**
> 
> But he assigned entirely unreal meanings to redshift-distance.
> 
> If it is actually "Expanding Universe" ... then yes ... it is Lemaitre Law.
> 
> But unfortunately ... there is no proof of expanding universe. Only modified translation is the proof. Theory had never reached to the correct interpretation of Hubble observation. In year 1930, both de-Sitter and Eddington were admitting this thing.
> 
> **Any fake theory could not replace this thing.**
> 
> I am an outside (of science community) observer.
> 
> Regards!
> 
> Khuram Rafique

**Comment by khuramonline** [comment]
Following is some background that may further explain the main topic.

Fritz Zwicky (so called father of dark matter) had proposed various alternative mechanisms to explain redshifts in those early days but he himself accepted that they did not work as expected.

Since all other ‘mechanisms’ failed … so only the ‘velocity’ mechanism survived.

But that was not just the survival of fittest thing.

Hubble was skeptical of velocity interpretation of redshifts **but** … actually he was also alone. He did not openly disregard velocity meanings. He cited de-Sitter in his 1929 paper that in the de-Sitter Cosmology, displacement of spectra arise from reasons other than motion.

But … de-Sitter also left him alone. Mainly because perhaps the specific point of de-Sitter was not having direct link with General Relativity.

**After 1929, de-Sitter and Arthur Eddington both were anxious how to explain Hubble type redshift-distance relationship within GR framework.**

Hubble was thinking that de-Sitter already had the prediction of cosmological redshifts in terms other than motion.

de-Sitter knew that he did have prediction but only by way of speculation or expectation out of observed data of few redshifted galaxies by time 1917.

**de-Sitter did speculate that redshift could be systematically linked with distance** and he had speculate this thing in year 1917.

http://www.pitt.edu/~jdnorton/teaching/2590_Einstein_2015/pdfs/de%20Sitter%20MNRAS%201917.pdf

Above is de-Sitter (1917) paper. The speculation about redshift-distance can be seen at the end of page No.26.

**He is saying that distance may cause redshifts , giving rise to spurious positive radial velocity.**

Meanings of **‘Spurious’** can be seen here:

[SPURIOUS | meaning in the Cambridge English Dictionary](https://dictionary.cambridge.org/dictionary/english/spurious)

[false](https://dictionary.cambridge.org/dictionary/english/false) and not what it [appears](https://dictionary.cambridge.org/dictionary/english/appear) to be, or (of [reasons](https://dictionary.cambridge.org/dictionary/english/reason) and [judgments](https://dictionary.cambridge.org/dictionary/english/judgment))[based](https://dictionary.cambridge.org/dictionary/english/based) on something that has not been [correctly](https://dictionary.cambridge.org/dictionary/english/correct) [understood](https://dictionary.cambridge.org/dictionary/english/understood) and [therefore](https://dictionary.cambridge.org/dictionary/english/therefore) [false](https://dictionary.cambridge.org/dictionary/english/false):

On the next page (27), he is saying that **Helium stars actually show this type of displacement of (spectra)** … On same page, he is using term **‘apparent velocities’ for galactic redshifts**.

Who was [Willem de Sitter - Wikipedia](https://en.wikipedia.org/wiki/Willem_de_Sitter)?

After development of GR equation in 1916 … Einstein was perhaps the first who developed a model of whole universe based on GR equations in year 1917.

de-Sitter was the second person who also developed a model of whole Universe based on GR equations in same year 1917.

The 1917 de-Sitter paper is like a ‘solution’ to GR equations or at least regarded as solution to GR equations.

**In this 1917 paper, de-Sitter is saying that distance may be linked to ‘spurious’ velocity (redshift)**.

In this paper … he did develop solution to GR equations … but there is lot of general discussion as well … like various developments in astronomy etc. including redshifts.

**Now whether had ‘Spurious Velocity’ come from GR or from general speculation?**

If it came from GR, **then why de-Sitter and Eddington … after 1929 … were sitting together to find a solution of Hubble type redshifts from within GR equations and were not reaching at satisfactory conclusion…???**

I do not know answer to this question. Only justification is that de-Sitter knew that page 26–28 of his 1917 paper were mere wild speculations and thus they did not relate to the current problem of how to account for Hubble type redshift-distance relationship within the framework of GR.

de-Sitter and Arthur Eddington were actually worrying in year 1930 regarding how to account for Hubble type redshift-distance relationship within the framework of GR equations. Following reference tells the story that they were worrying on this point:

http://www.physics.umd.edu/grt/taj/675e/Luminet_on_Lemaitre_history.pdf

**The story can be found on page 8 of this PDF file (PDF page).**

The rest of the story is that **Lemaitre approached Eddington that he already had published similar solution in year 1927.**

**Then Eddington got a plan. He advised Lemaitre to publish the translation but with modifications.**

The **English translation (1931)** of Lemaitre paper **changed the game**. That was a modified and deceptive translation … **showing as if Lemaitre had already (in 1927) derived Hubble Law from GR equations**.

In this way, Hubble was left alone. **de-Sitter also adopted Lemaitre explanation**. Zwicky proposed alternatives but then withdrew his proposals not because he was satisfied with expansionism … only because he had not found viable alternative.

**About Latest developments:**

It is often stated that latest developments have confirmed expansion of universe. Actually there is no latest development with regards to providing direct evidence that galaxies are in fact in motion. Even the so called ‘inflation theory’ has been ‘derived’ from de-Sitter model where cause of redshifts was other than motion.

**Reply by khuramonline** [reply]
Edwin Hubble was alone as skeptic of velocity meaning of redshift. Till the end he tried to tell that velocity was not the meaning of redshift. 

Meaning of redshifts according to Edwin Hubble:
Following is from book “The Realm of the Nebulae” (1936)

![image](https://github.com/a-cosmology-group/acg/assets/132253587/a5c5e4a4-42b6-4399-9826-f713a836becb)

Here Edwin Hubble is saying that redshifts can be provisionally explained in terms of ‘velocities’ but ultimate meanings should come from observations.

Then according to Newspaper December 31, 1941, Edwin Hubble finally sorted out that galaxies were not in motion.

[Hubble: No Evidence of ‘Big Bang’ Theory](https://ladailymirror.com/2011/12/31/hubble-no-evidence-of-big-bang-theory/) (https://ladailymirror.com/2011/12/31/hubble-no-evidence-of-big-bang-theory/)

![image](https://github.com/a-cosmology-group/acg/assets/132253587/7a546ab9-9e6f-4f66-9cd7-594ebe017e61)

**Reply by HanDeBruijn** [reply]
Very interesting, especially the newspaper article and writeup by Edwin Hubble himself !!

**Reply by khuramonline** [reply]
About screenshot from book “The Realm of the Nebulae” (1936)

![image](https://github.com/a-cosmology-group/acg/assets/132253587/4bd0f42f-0f59-4ec0-b920-b121f681c22f)

There are important points. Here Edwin Hubble is actually saying that same velocity "scale" may be used for redshifts because structure wise the displacement of spectral lines were same to velocity interpretation. Since actual meaning of cosmological redshift was not known so at least there should be any scale to measure the quantity of redshift. Hubble is only saying that same "velocity" scale can be employed until the right meaning is known.

He also emphasized that adjective "apparent" be used before the term "velocity" and wherever the adjective is missing, it should be understood for granted that the adjective "apparent" is not missing.

**Reply by HanDeBruijn** [reply]
@khuramonline : May I ask you what your thoughts are about the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) ? Would that be a viable alternative ?

**Reply by khuramonline** [reply]
This variable mass theory seems to be a variation of steady state theory. And steady state theory is already an expanding universe theory. If redshift is due to increase in mass then expansion is due to what?

Here expansion is going on without redshifting. Because redshifts are due to that mass is variable.

Steady State Thorists also think (wrongfully) that Hubble had discovered the expanding universe.

**Reply by HanDeBruijn** [reply]
Quote from a page on my website: _[Length Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm) is observationally equivalent with the Expansion of Empty Space_.

**Comment by khuramonline** [comment]
It is known but not commonly known that there were certain discrepancies between  Lemaître (1927) French article and 1931 English translation of same article. Original 1927 paper had failed to impact anything. However during 1930 or 1931,  Lemaître sent original 1927 French article to Arthur Eddington. Actually  Lemaître had already sent this paper to Eddington in 1927 as well but at that time Eddington had not responded. By 1930, Hubble Law of redshift-distance was already surfaced.

Now when in 1930 or 1931 Lemaître resent his 1927 French paper to Eddington, this time Eddington responded him. At that time Eddington was in search of any solution of GR where Hubble Law could be explained.

After reading the French paper, Eddington advised Lemaître to translate the paper in English **but with certain modifications** -- for the purpose of publication in a renowned journal that was under the administrative control of Eddington himself.

Then Lemaître translated the paper in year 1931 and omitted crucial points from translation.

[This is link](https://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1927ASSB...47...49L&defaultprint=YES&filetype=.pdf) to the original 1927 French article:

Following is screenshot from 1927 French article. There is complete paragraph under equation No.23 and in that paragraph **there is reference to Hubble**.

![image](https://github.com/a-cosmology-group/acg/assets/132253587/0d63f16b-4a6e-4133-8781-c474b2491a91)

And this is [link to](https://watermark.silverchair.com/mnras91-0483.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAsgwggLEBgkqhkiG9w0BBwagggK1MIICsQIBADCCAqoGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMmRscbrhghw59Thd6AgEQgIICe9px8sKWM-Brad5WtVupcqOBSTSxFOplwWHuhqa58yWKnXrTXKCNtu0DNfaVyP_OqE8Gk8VcZTO-qOLphIARl6KZ4_RbGplCNSnII6ZcvhupUknZNS4-R4iM66EUnpeI-i6fC1mOtlwntxgCIjdXltEZONodU01tp-J1B5qpw-BR0Lc5lDF32fimrwAzgXT6fV9nBizCAI9w9Kc2H77KbpvVcvBM3BjehT0R8KkJUTgRlLjs82jFtWA63v62K9u9qfDbvAcm7o5MBY10kkWJ4BhcuiK8lNa1uTCu4N_13sD6fBNp9X2ZgYauHYsLR7Q571s2hGP5wGIVf3W-9czKx8pZstm9ANCSn9yDdjcMBRBsFAMd_OOye3_SPPObHkNAsVpGAUfpq8ThfD3yUy0lF5DlgWu8UuLtRVCrttl0dq0mH-8Uk2H_DL9u5XZqhtCUmJ4g5f8ZIl7S3Ha80efJ9gUcUDe4IgbJc8Gb5_nCyznPhj8tmGa0z2z3gKpZjrce-tgw1ipKCa-AjonuE_GYXOHUMulkjf9bPyCI5_S8wgnsr83GasH4Q8qmd2pCjQm1ubPuSnqRqSF0v7r8KXHfFKOBXrEYzbONavC3YLRJ_Jw-LF6lUfW-VeQy9TtFQL1VG4k9uqyZ50xu01ei7bYiuAFxYMW0y0IlaYUDRUnbz2XvuU5Gt-ziqBM9koptPIJDu1IuOTj1RejIpO5Pv3xV3gqfarwCAee0nqfgHgXriU0SDitdLwD65Wll-IFn4wOAvrawZmgX0ozEeAqIXJwNxD7j-RRh8LeK0XG6J6L_9bxmAOf3jKos1HKdMWlDgUUc4gfgr7PFjdVanINB) Lemaître's 1931 translation of same 1927 French article.

Following is screenshot from English Translation (1931):

![image](https://github.com/a-cosmology-group/acg/assets/132253587/40035cf4-4674-43e0-bd97-fa08857629aa)

Please note that whole paragraph after equation No.23 is replaced by just one sentence and **thus reference to Hubble has been totally omitted.**

And following is the screenshot from a secondary source where they have translated the paragraph under equation No.23:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/05c53247-0c42-4896-9357-2aa0562083bb)
 
Position is that Hubble had worked out the method of finding distances of galaxies based on redshifts in year 1926. 

In year 1927, Lemaitre had data of redshifts and he duly cited Hubble as source of method of finding distances of galaxies using redshifts. In this way, based on Empirical Data, Lemaitre reached to Hubble's Law before Hubble.

In year 1929, Hubble announced his redshift-distance relationship.

In year 1931, Lemaitre published English translation of his 1927 article. In the translation, he omitted the reference to Hubble.

**The translated article (1931) deceptively showed that Lemaitre had already derived Hubble Law from GR equations.** 

Thus Expanding Universe Theory only has this deceptive relationship with GR.

**Reply by ftessier** [reply]
@marmetl perhaps we need a History section in the discussions? Or perhaps better a `history` label, since there may be interesting historical discussions in all sections.

**Reply by RedshiftDrift** [reply]
The ":bulb: Metaphysics and Philosophy" Category was intended for that
`Space, time, reality, existence, scientific method, history of cosmology, infinity, creation, etc`

'History' is interesting and useful, and Andre Assis and others have contributed many papers on that topic.
However, in earlier discussion groups 'history' never became a major topic, so we can wait and see before adding another `Category`.

For now this discussion can be moved to the proper category since it does not discuss `Redshift models` but the `history of redshift models`.
